cmd/ooniprobe/internal/nettests/nettests.go

```diff
@@ -94,7 +93,7 @@ func (c *Controller) BuildAndSetInputIdxMap(testlist []model.OOAPIURLInfo) ([]st
 	urlIDMap := make(map[int64]int64)
 	for idx, url := range testlist {
 		log.Debugf("Going over URL %d", idx)
-		urlID, err := c.Probe.DB().CreateOrUpdateURL(
+		urlID, err := c.Probe.WriteDB().CreateOrUpdateURL(
```

Comment:
Same as above with a minor clarification.

In the future, when we will want to unify the `nettests` package with code in the `internal` package, we will need to perform an additional modification, i.e., making `nettests` depend on a `WritableDatabase`. I do not think we need to worry about doing this change right now (it will be a bit messy to do).

However, for this diff we should replace `c.Probe.WriteDB` with `writableDB(c.Probe)` where:

```Go
function writableDB(probe ooni.ProbeCLI) model.WritableDatabase {
  return probe.DB()
}
```

This adapter will _ensure_ that we can write code in terms of `WritableDatabase`.